src/FEMMDeforLinearBaseModule.jl

```diff
@@ -92,6 +93,47 @@ function mass(self::FEMMDeforLinearAbstract,  geom::NodalField{FFlt},  u::NodalF
     return mass(self, assembler, geom, u);
 end
 
+function damping_ABC(self::FEMMDeforLinearAbstract, assembler::A,
+                     geom::NodalField{FFlt}, u::NodalField{T},
+                     impedance::T, surfacenormal::SurfaceNormal) where {A<:SysmatAssemblerBase, T<:Number}
+    fes = self.integdata.fes
+    nfes = count(fes);
+    DoFperNode = ndofs(u);
```

Comment:
It may be a good idea to stick to nomenclature established in other FEMM implementations:
```
# Constants
    nfes = count(integdata.fes); # number of finite elements in the set
    ndn = ndofs(u); # number of degrees of freedom per node
    nne = nodesperelem(integdata.fes); # number of nodes for element
    sdim = ndofs(geom);            # number of space dimensions
    mdim = manifdim(integdata.fes); # manifold dimension of the element
    Kedim = ndn*nne;             # dimension of the element matrix
```